src/components/icon/styled.ts

```diff
@@ -2,11 +2,10 @@ import styled from 'styled-components';
 import SvgIcon from '@fellesdatakatalog/icons';
 
 const Icon = styled(SvgIcon)`
-  width: 22px;
-  height: 22px;
-
+  width: 24px;
+  height: 24px;
   & * {
-    stroke: #fff;
+    stroke: #335380;
```

Comment:
Finnes ikke fargen i theme.colour?

Reply:
Pooria har planlagt å bruke fargene fra fdk-theme, men legger de inn på et senere tidspunkt, så jeg tenkte å følge samme fremgangsmåte for å unngå rot.